opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -226,79 +258,69 @@ message HistogramValue {
   // compact encoding on the wire.
   // If the value of 0 occurs for the first data point in the timeseries it means that
   // the timestamp is unspecified. In that case the timestamp may be decided by the backend.
+  // Note: this field is always unspecified and ignored if MetricDescriptor.type==GAUGE_HISTOGRAM.
   sfixed64 start_time_unixnano = 1;
```

Comment:
Nit: I am not sure if I know of an usecase where this is not the same for all points, just to document this.

Reply:
Good to know. If that is the case then perhaps this can be moved to HistogramTimeSeries. The same change can be done for all other value types, so let's consider it in a separate PR. I added an issue https://github.com/open-telemetry/opentelemetry-proto/issues/38

Reply:
FYI: We need some more investigations not just benchmarks for that change.